packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -315,7 +315,10 @@ export function _SignInStart(): JSX.Element {
                     />
                   </Form.ControlRow>
                   <InstantPasswordRow field={passwordBasedInstance ? instantPasswordField : undefined} />
-                  <Form.SubmitButton hasArrow />
+                  <Form.SubmitButton
+                    hasArrow
+                    sx={t => ({ marginTop: t.space.$2 })}
```

Comment:
Why are we adding margin to these screens only?

Reply:
We should add to all the screens of the SignIn/SignUp. This is the spacing the 'Continue' primary button has from the inputs. I removed some weird spacing from the inputs and now we need to add this to match the designs.

Reply:
Yeah I think this can be solved with adding a gap, I've done that in most screens in the SignIn/SignUp so I could increase that gap.

Reply:
Removed!